pytorch_pfn_extras/_cupy/cupy.py

```diff
@@ -0,0 +1,2 @@
+class ndarray:
+    pass
```

Comment:
Could you rename this file to `_cupy_stub.py`? I think it's better to avoid filenames that conflicts with the real module name.